owlkettle/widgets.nim

```diff
@@ -3459,10 +3459,9 @@ renderable Scale of BaseWidget:
       state.value = state.internalWidget.gtk_range_get_value().float64
   
   hooks marks:
-    (build, update):
-      state.internalWidget.gtk_scale_clear_marks()
-      let hasScaleMarks = widget.hasMarks and widget.valMarks.len > 0
-      if hasScaleMarks:
+    property:
+      if widget.hasMarks:
```

Comment:
property hooks should never access has/valMarks. Use state.marks instead,

Reply:
Done-ish. Basically I just removed the if-check which makes it as correct as it'll get.